.release.cloudbuild.yaml

```diff
@@ -141,10 +141,10 @@ steps:
   waitFor: ['PullMetadataEnvoy']
 
 - name: 'gcr.io/cloud-builders/docker'
```

Comment:
Let's move all these entries to a separate cloudbuild.yaml file in the /third-party directory so that it's not triggered with every release.

Reply:
I don't quite understand why we have these entries. They are copying already built images from gcr.io/ml-pipeline/... to gcr.io/ml-pipeline/google/pipelines/...:$TAG_NAME (released tag name).

But I couldn't find any usage of gcr.io/ml-pipeline/google/pipelines, our lite deployment only references gcr.io/ml-pipeline/image-name.

Can you explain what these are for? I didn't understand so I merely changed the tags to my new tags.

Reply:
I also checked https://github.com/kubeflow/manifests, they are also not using these. @IronPan do you know what these are for? Just meant as taking a snapshot of the release artifacts?

Reply:
@bobgy no need to add license-compliance part in the tag here. As long as we're building these default ones with the source and license included, we're good.

Reply:
@neuromage My reasoning of the extra `license-compliance` part: https://github.com/kubeflow/pipelines/pull/2201#issuecomment-535403652

It's for purpose of safe gradual release, because images like `gcr.io/ml-pipeline/argoexec:v2.3.0` are already in use today. If I push a new image to the same tag, everyone will switch to that new image immediately.

Technically, there's no difference between previous and new images. New images also just passed our presubmit tests. So risk pushing new images under the same tag is small too. I think we can also choose to do that.

WDYT? I'd like to double check if you want me to push new images under the same tag? or would you want to suggest a simpler naming?

Reply:
I think if we're distributing the images, we should just version them (since we technically rebuild these every release). Ideally, we would name them something like `argoexec-v2.3.0-kfp:0.1.30` which can indicate that this is based of argo v2.3.0, but released under kfp with tag 0.1.30. I don't want to block this PR, but can we consider using this convention? This can be done in a future PR. 

/cc @IronPan

Reply:
>`argoexec-v2.3.0-kfp:0.1.30`
Should we do that even if we only release a modified version of the image once per Argo release (not once per KFP release)?

Using different tags for the same image might be a bit confusing.